prusti-common/README.md

```diff
@@ -0,0 +1,3 @@
+# prusti-common
+
+Common utilities used across the Prusti project.
```

Comment:
I would add "compiler-agnostic" (utilities), as the idea is to never use rustc as a library in this crate.